### PR TITLE
Update docker-compose configurations and readme

### PR DESCRIPTION
**Pull request by nuest**
Finally only one compose file, and generic compose configuration now available at https://github.com/o2r-project/reference-implementation

This repo only has one compose configuration for platform development (./client is mounted into a webserver)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ _Leveraging reproducible research_
...
+
+```bash
+docker build --tag platform .
+
+docker run -d -p 80:80 platform
+```
+
 ## Configure
 
 Create a copy of the file `client/app/config/configSample.js` and name it `client/app/config/config.js`. You must configure the required application settings in this file, which is not part of the version control:
```
Comment (MarkusKonk) [inline README.md]: So far, it also worked if I deleted the config as there are also predefined settings.

File: README.md [hunk 2]
```diff
@@ -34,112 +42,64 @@ window.__env.userLevels.regular = /*Integer containing the required user level f
...
+This project contains one `docker-compose` configuration (file `docker-compose.yml`) to run all microservices & databases, and mount the client application directly from the source directory `client`.
 
-#### Required settings
+_The client must be build on the host!_
 
-Some of the settings to run the platform cannot be published. These must be provided at runtime using envionment variables as is described in the OS-specific instructions below. Not providing one of these paramters results in untested behaviour.
+**Required settings**
 
+Some of the settings to run the platform cannot be published.
```
Comment (MarkusKonk) [inline README.md]: Refer to git ignore?

File: README.md (hunk 1 above)
Comment (MarkusKonk) [inline README.md]: So far, it also worked if I deleted the config as there are also predefined settings.

Comment (nuest) [reply README.md]: That you be for you and @jansule to change, as I am not an expert in the AngularJS app.

File: README.md (hunk 2 above)
Comment (nuest) [reply README.md]: I do not understand. Do you mean to add these settings to a file which is already included in the gitignore?

Comment (jansule) [reply README.md]: I think there is a misunderstanding. The text does not refer to settings stored in a file but as environment variables. Therefore gitignore does not need to be changed.

**Pull request merged**